src/components/FloatingBar/FloatingBar.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+
+import classNames from 'classnames'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import { Loader } from 'decentraland-ui/dist/components/Loader/Loader'
+
+import { forumUrl } from '../../entities/Proposal/utils'
+import useFormatMessage from '../../hooks/useFormatMessage'
+import useProposalComments from '../../hooks/useProposalComments'
+import Link from '../Common/Typography/Link'
+import Forum from '../Icon/Forum'
+import Open from '../Icon/Open'
+import Reactions from '../Icon/Reactions'
+
+import './FloatingBar.css'
+
+interface FloatingBarProps {
+  isVisible: boolean
+  showViewReactions: boolean
+  scrollToComments: () => void
+  scrollToReactions: () => void
+  proposalId: string
+  discourseTopicId: number
+  discourseTopicSlug: string
+}
+
+const FloatingBar: React.FC<FloatingBarProps> = ({
+  proposalId,
+  discourseTopicId,
+  discourseTopicSlug,
+  showViewReactions,
+  isVisible,
+  scrollToComments,
+  scrollToReactions,
+}) => {
+  const t = useFormatMessage()
+  const { comments, isLoadingComments } = useProposalComments(proposalId)
+  if (!isVisible) return null
```

Comment:
Maybe we could use this prop to add/remove a css class so it can have an opacity transition